ui/StatusQ/tests/TestControls/tst_test-StatusInput.qml

```diff
@@ -7,26 +7,76 @@ import StatusQ.Controls.Validators 0.1
 import StatusQ.TestHelpers 0.1
 
 Item {
+    id: root
+
     width: 300
     height: 100
 
-    property int _defaultValidationMode
+    Component {
+        id: defaultComponent
+
+        StatusInput {
+            label: "Control under test"
+            charLimit: 30
+            placeholderText: `Must match regex(${validators[0].regularExpression.toString()}) and <= 30 chars`
+            focus: true
+
+            validators: [
+                StatusRegularExpressionValidator {
+                    regularExpression: /^[0-9A-Za-z_\$-\s]*$/
+                }
+            ]
+        }
+    }
+
+    property QtObject dataObject: QtObject {
+        property string text: "Test text"
+        property string emoji: "👍"
+        property string color: "#FF0000"
+    }
+
+    Component {
+        id: bindedTextComponent
+
+        StatusInput {
+            charLimit: 10
+            input.isIconSelectable: true
+            placeholderText: qsTr("Enter an account name...")
+            text: dataObject.text
+            input.asset.emoji: dataObject.emoji
+            input.asset.color: dataObject.color
+
+            focus: true
+
+            validators: [
+                StatusMinLengthValidator {
+                    errorMessage: qsTr("You need to enter an account name")
+                    minLength: 1
+                },
+                StatusRegularExpressionValidator {
+                    regularExpression: /^[^<>]+$/
+                    errorMessage: qsTr("This is not a valid account name")
+                }
+            ]
+        }
+    }
+
+    Loader {
```

Comment:
An alternative to manually manage test components creation is to use [createTemporaryObject](https://doc.qt.io/Qt-5/qml-qttest-testcase.html#createTemporaryObject-method). This way we can remove the loader.